packages/tutanota-utils/lib/ArrayUtils.ts

```diff
@@ -512,4 +515,13 @@ export async function partitionAsync<T>(array: Array<T>, predicate: (arg0: T) =>
 	const mask: Array<boolean> = await Promise.all(array.map(predicate))
 	const [left, right] = partition(zip(mask, array), item => item[0])
 	return [left.map(i => i[1]), right.map(i => i[1])]
+}
+
+/**
+ * Create an array with n elements by calling the provided thunk
```

Comment:
Thunk is generally something that doesn't accept anything but whatever

Reply:
![Screenshot from 2022-02-28 16-26-42](https://user-images.githubusercontent.com/22109738/156009886-72a8ad22-0913-426c-aaf7-5fac9bbdab78.png)